katas/langtonant/percy/antMovement.test.js

```diff
@@ -0,0 +1,23 @@
+import { movements } from './ant';
+
+test('langtonAnt movements', () => {
```

Comment:
This test looks good, as next step, please also add tests for the different functionality units you have in your code, for example, functions `newMovement`, `color` and main kata: given a number of steps, validate final status of the grid